<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignWindow</class>
 <widget class="QWidget" name="SignWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create an account</string>
  </property>
  <widget class="QPushButton" name="SignButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Sign up</string>
   </property>
  </widget>
  <widget class="QLabel" name="PasswdLabel1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Input password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswdEdit1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>49</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Please input password</string>
   </property>
  </widget>
  <widget class="QLabel" name="PasswdLabel2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Input again:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswdEdit2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Please input again</string>
   </property>
  </widget>
  <widget class="QLabel" name="AccountLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>User name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AccountLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Please input your name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
